modules/database/src/ioc/rsrv/camsgtask.c

```diff
@@ -163,6 +163,8 @@ void camsgtask ( void *pParm )
 
 int casClientInitiatingCurrentThread ( char * pBuf, size_t bufSize )
 {
+    if ( ! rsrvCurrentClient )
+        return RSRV_ERROR; /* not yet initialized, or disabled via dbServer */
```

Comment:
Breaks the build on VS-2012 and older, this adds a statement before declarations inside a block.